azext_iot/iothub/_help.py

```diff
@@ -86,3 +86,57 @@ def load_iothub_help():
           text: >
             az iot hub job cancel --hub-name {iothub_name} --job-id {job_id}
     """
+
+    helps["iot pnp twin"] = """
+        type: group
+        short-summary: Manipulate and interact with the digital twin of an IoT Hub device.
+    """
+
+    helps["iot pnp twin invoke-command"] = """
```

Comment:
is it called a command in iot pnp? or direct-method

Reply:
its called a command in PnP world